src/_utils/itineraryCollapsible/story.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
```

Comment:
We shouldn't have stories for internal components. They're exposed though `Itinerary`